packages/devtools_app/lib/src/shared/console/widgets/description.dart

```diff
@@ -554,13 +553,23 @@ class DiagnosticsNodeDescription extends StatelessWidget {
 }
 
 class DescriptionDisplay extends StatelessWidget {
-  const DescriptionDisplay({
+  DescriptionDisplay({
     super.key,
     required this.text,
     this.multiline = false,
     this.actionLabel,
     this.actionCallback,
-  });
+  }) {
```

Comment:
these can be part of the constructor definition so we don't have to have a body and this can stay const:
```
  ...
  this.actionCallback,
}) : assert(multiline ? actionLabel == null : true, 'Action labels are not supported for multiline descriptions',), 
  assert((actionLabel == null) == (actionCallback == null), 'Actions require both a label and a callback');
```

Reply:
Oh nice, done!